.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+documentation:
+  - changed-files:
+      - any-glob-to-any-file: ['docs/**', 'mkdocs.yml']
+docker:
+  - changed-files:
+      - any-glob-to-any-file: ['**/Dockerfile', 'docker-compose.yml']
+element:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/elements/**']
+SQL:
+  - changed-files:
+      - any-glob-to-any-file: ['**/*.sql']
+python:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/python/**']
```

Comment:
Want to include `requirements.txt` (or `images/plbase/python-requirements.txt`, which should really be renamed)?

Reply:
I feel like there are diminishing returns for some of these one-of files.

Reply:
The Python deps change as often, if not more so, than most of our dependencies. Seems pretty low-effort/high-reward to add this if we want accurate labels.

You may also want to add `apps/prairielearn/elements/**/*.py` to this list.

Reply:
Added